docs/dev/reference/sdks/docommand.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Implement or use DoCommand"
+linkTitle: "DoCommand"
+weight: 67
+type: "docs"
+description: "Implement DoCommand in your module or use it from Viam's SDKs."
+tags: ["sdk", "extend"]
+date: "2025-04-29"
+# updated: ""  # When the content was last entirely checked
+---
+
+The `DoCommand` method is a flexible wrapper that you can use to send commands that don't fit into any other method.
```

Comment:
```suggestion
The `DoCommand` method is a flexible wrapper that you can use to send commands that have no corresponding built-in API method.
```
Maybe something like this? I don't like the "don't fit into any other method" because i think it's odd to think of commands as inside methods